README.md

```diff
@@ -4,6 +4,7 @@ the [Web Applications Working Group](https://www.w3.org/2019/webapps/).
 ## Useful links
 * [Explainer](https://github.com/w3c/manifest/blob/gh-pages/explainer.md)
 * [The Web Application Manifest specification](https://www.w3.org/TR/appmanifest/)
+* [Manifest incubations](https://github.com/WICG/manifest-incubations)
```

Comment:
add https://wicg.github.io/web-share-target/ too?

Reply:
It's linked to already from the incubations, so we probably don't need to add it here.